tests/processes/ProcessingTest.py

```diff
@@ -399,6 +399,17 @@ def test_yield_ignore_ranges(self):
             self.assertEqual(test_source_range.end.line, 2)
             self.assertEqual(test_source_range.end.column, 43)
 
+        test_file_dict_a = {'f':
+                            ('# noqa aBear\n',
```

Comment:
yeah, `# noqa` should get applied on any bear. BTW, with this example, there's no *intuitive* way to stop ignore? using `# stop ignoring` doesn't make sense when we start with `# noqa`. I thought we agreed that `# noqa` will be the one-line equivalent for start ignoring?

but IMO we could allow for bear params; if no params then match everything